CONTRIBUTING.md

```diff
@@ -192,6 +205,8 @@ In order to backport changes to these branches the process we use is as follows:
 
 Any other changes to the release branches should follow the normal 2-ACK merge policy.
 
+## Merging
```

Comment:
In fd36e349f3cf21683421d0eae3808d439d5a2c51:

Looks like you added this section but did not fill it in..

Reply:
I think you still mean to fill this in.  Probably stick this in draft since I don't think you want this considered for merge.